### DISCUSSION TITLE
Advanced Portal Install

### DISCUSSION THREAD

**Discussion by MCancellier** [jprochazka/adsb-receiver#614] [Q&A]
Could we get a little detail on how to set up the advanced version of the portal? I'm still not sure how to answer the prompts.

**Comment by jprochazka** [comment]
Some documentation would be a nice addition. 

I have added a page to the wiki which can be found at the following address.
https://github.com/jprochazka/adsb-receiver/wiki/Advanced-Portal-Installation-Guide-(PHP-Portal)

Let me know if it is helpful or not or needs additional details.
